src/plone/api/tests/test_content.py

```diff
@@ -376,6 +376,46 @@ def test_create_at_with_title_in_request(self):
 
         self.assertEqual(page.title, 'Test document')
 
+    @unittest.skipIf(HAS_PACONTENTYPES, 'Archetypes only')
+    def test_create_at_event(self):
```

Comment:
@hvelarde why do not you add this test to the package? It will prevent #364  from occurring again.

Reply:
Archetypes is going to die soon.